charts/ethos-repl-docmosis-backend/values.yaml

```diff
@@ -1,7 +1,7 @@
 java:
   image: 'hmctspublic.azurecr.io/ethos/repl-docmosis-backend:latest'
   applicationPort: 8081
-  ingressHost: ethos-repl-docmosis-backend-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+  ingressHost: ethos-repl-docmosis-backend-{{ .Values.global.environment }}.platform.hmcts.net
```

Comment:
change should be reverted. the platform change was only needed for preview environments. same for the changes below as well and in other files